components/gitpod-cli/pkg/supervisor-helper/status.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package supervisor_helper
+
+import (
+	"context"
+
+	supervisor "github.com/gitpod-io/gitpod/supervisor/api"
+)
+
+func GetWorkspaceResources(ctx context.Context) (*supervisor.ResourcesStatusResponse, error) {
+	conn, err := Dial(ctx)
+	if err != nil {
+		return nil, err
+	}
+	client := supervisor.NewStatusServiceClient(conn)
```

Comment:
In the future, we could expand supervisor to also be aware of the vm resources and return additional metadata. As mentioned [here](https://github.com/gitpod-io/website/pull/2207/files#r894121494), the available amounts might not be true, if the vm is running out of resources due to noisy neighbours.

Reply:
Let's stick with the simple approach for now and assume that VM always has enough resources.

Reply:
yep, that's the plan 👍